az-exec-util/build.gradle

```diff
@@ -17,6 +17,7 @@ apply plugin: 'c'
 model {
     components {
         main(NativeExecutableSpec) {
+            targetPlatform "osx_x86-64"
```

Comment:
Is this also a TODO? I'm not expert with Gradle, but does this sett the target platform regardless of what the current platform is?

Reply:
I don't know much about grade.  all the Gradle docs state is "Specifies the names of one or more [Flavor] that this component should be built for."

I think this forces the build to x86 vs ARM so that it runs Azkaban in rosetta mode on ARM / apple silicon.

Reply:
I now see it only applies to a "c" utility in `az-exec-util`.  I think it just makes the utility written in C build for x86, so the util will run in rosetta mode on ARM.
There may be a way to get that utility written in C to work on ARM, but that was beyond the scope of what I was able to pull together for this PR.

Reply:
Thanks. I was wondering what happens on other than osx platforms. Seems that this doesn't ruin it for them, at least the Travis build seems to have succeeded past the point of building `az-exec-util`.

Reply:
you are welcome to fork this and create a different MR if you can get everything else to work.  I just put this MR here for posterity for anyone else having apple silicon / ARM issues.